examples/oneapi/cpp/source/graph/graph_service_functions.cpp

```diff
@@ -28,19 +28,19 @@ using namespace oneapi::dal::preview;
 int main(int argc, char **argv) {
     const std::string filename = get_data_path("graph.csv");
 
-    graph_csv_data_source ds(filename);
-    load_graph::descriptor<> d;
-    auto my_graph = load_graph::load(d, ds);
+    const graph_csv_data_source ds(filename);
+    const load_graph::descriptor<> d;
+    const auto my_graph = load_graph::load(d, ds);
     std::cout << "Number of vertices: " << get_vertex_count(my_graph) << std::endl;
     std::cout << "Number of edges: " << get_edge_count(my_graph) << std::endl;
 
-    auto node_id = 0;
+    std::int32_t node_id = 0;
```

Comment:
And additionally change "node" terminology to "vertex"
Please also check if it is used in some other code